p2p/gossip/protocol.go

```diff
@@ -202,6 +202,16 @@ func (prot *Protocol) getPriority(protoName string) priorityq.Priority {
 	return v
 }
 
+func (prot *Protocol) drainPropagationQueue() {
```

Comment:
is this really needed? please revise...

Reply:
probably not, was a desperate attempt to fix travis.. going to remove now.